src/commands/cmd_explain.c

```diff
@@ -49,12 +50,12 @@ int MGraph_Explain(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     if (AST_PerformValidations(ctx, ast) != AST_VALID) return REDISMODULE_OK;
 
     if (ast->indexNode != NULL) { // index operation
-        const char *strPlan = Index_OpPrint(ast->indexNode);
-        RedisModule_ReplyWithSimpleString(ctx, strPlan);
+        char *reply = (ast->indexNode->operation == CREATE_INDEX) ? "Create Index" : "Drop Index";
```

Comment:
What happened to Index_OpPrint?

Reply:
I deleted it so that I could remove the `ast.h` dependency from the index files!